apple/RNCWebViewManager.m

```diff
@@ -199,6 +200,18 @@ - (RCTUIView *)view
   }];
 }
 
+RCT_EXPORT_METHOD(injectJavaScriptWithWebViewKey:(nonnull NSString *)webViewKey script:(NSString *)script)
+{
+  dispatch_async(dispatch_get_main_queue(), ^{
+    NSMutableDictionary *sharedWKWebViewDictionary = [[RNCWKWebViewMapManager sharedManager] sharedWKWebViewDictionary];
+    WKWebView *wkWebView = sharedWKWebViewDictionary[webViewKey];
+
+    if (wkWebView != nil) {
```

Comment:
if `wkWebView` is nil, maybe add a log entry `RCTLogError(@"Invalid view returned from registry, expecting RNCWebView, got: %@", view);`